lib/routes/account.js

```diff
@@ -498,12 +498,14 @@ module.exports = function (
               // If the account is already verified, they may be e.g.
               // clicking a stale link.  Silently succeed.
               if (account.emailVerified) {
+                log.activityEvent('account.already_verified', account.uid, request)
                 return true
               }
               if (!butil.buffersAreEqual(code, account.emailCode)) {
                 throw error.invalidVerificationCode()
               }
               log.event('verified', { email: account.email, uid: account.uid, locale: account.locale })
+              log.activityEvent('account.verified', account.uid, request)
```

Comment:
Yeah, it's a bit of a mess, see additional discussion in https://github.com/mozilla/fxa-auth-server/pull/947#issuecomment-110163835

We need to finish up the lifecycle event cleanup/notification work, but it never manages to get to the top of a priority stack :-(